samples: bluetooth: mesh: sensor sample features (#8783)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/sensor_client/README.rst

   Can be used to input the OOB authentication value during provisioning.
   All buttons have the same functionality during the provisioning procedure.

> After the provision procedure is finished, the functionality of the buttons will be as follows:
> 
> Button 1:
>    Sends a setting get message.
> 
> Button 2:
>    Sends a setting set message, switching between the ranges specified in the :c:var:`temp_ranges` variable.
> 
> Button 3:
>    Sends a descriptor get message, requesting info about the :c:var:`bt_mesh_sensor_present_dev_op_temp` sensor.
> 

Reviewer: ```suggestion
Once the provisioning procedure has completed, the buttons will have the following functionality:

Button 1:
   Sends a get message for a setting.

Button 2:
   Sends a set message for a setting, switching between the ranges specified in the :c:var:`temp_ranges` variable.

Button 3:
   Sends a get message for a descriptor, requesting information about the :c:var:`bt_mesh_sensor_present_dev_op_temp` sensor.

```
